docs/user-guide/command-line-interface.md

```diff
@@ -81,6 +81,7 @@ Miscellaneous:
   --init                         Run config initialization wizard - default: false
   --env-info                     Output execution environment information - default: false
   --no-error-on-unmatched-pattern  Prevent errors when pattern is unmatched - default: false
+  --exit-on-fatal-error          Exit with exit code 2 in case of fatal error.  - default: false
```

Comment:
This should be synced with the description in `options.js`

(this whole block should be the same as the output from `eslint -h`).